documentation/jetty-documentation/pom.xml

```diff
@@ -9,12 +9,14 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>jetty-documentation</artifactId>
   <name>Documentation :: Guides</name>
-  <packaging>jar</packaging>
+  <packaging>pom</packaging>
 
   <properties>
     <bundle-symbolic-name>${project.groupId}</bundle-symbolic-name>
     <asciidoctor.skip>${skipTests}</asciidoctor.skip>
     <jacoco.skip>true</jacoco.skip>
+    <javadoc.skip>true</javadoc.skip>
```

Comment:
it's a pom so it's probably not needed (but definitely a nit comment :))

Reply:
This skips the internal classes from having a -javadoc.jar produced (it produced it, even as pom packaging).
It had the examples / demos used in the documentation.